src/amo/components/SectionLinks/index.js

```diff
@@ -90,17 +82,6 @@ export class SectionLinksBase extends React.Component<InternalProps> {
       );
     }
 
-    // See: https://github.com/mozilla/addons-frontend/issues/8680
-    if (!_config.get('enableFeatureRemoveSearchTools')) {
```

Comment:
@willdurand do we need the codes below this ?

Reply:
no, only code when the condition checks that `enableFeatureRemoveSearchTools` is `true`.